src/modules/iotjs_module_buffer.c

```diff
@@ -82,6 +82,20 @@ size_t iotjs_bufferwrap_length(iotjs_bufferwrap_t* bufferwrap) {
 }
 
 
+bool iotjs_jbuffer_is_bufferwrap(const jerry_value_t jbuffer) {
+  if (jerry_value_is_object(jbuffer)) {
+    void *native_p;
+    const jerry_object_native_info_t *type_p;
+    bool has_p = jerry_get_object_native_pointer (jbuffer, &native_p, &type_p);
```

Comment:
The name is a bit confusing, because `_p` suffix is used for pointers, but `has_p` is a boolean. What about `has_native_pointer`?